nginx.conf

```diff
@@ -104,6 +104,16 @@ http {
     
       proxy_pass $bucket_url;
       proxy_intercept_errors on;
+
+      # Cloudfront forwards these but doesn't consider them part of the cache key
+      # so drop them before passing to the origin, since responses aren't actually cacheable
+      proxy_set_header If-Match "";
+      proxy_set_header If-None-Match "";
+      proxy_set_header If-Modified-Since "";
+      proxy_set_header If-Unmodified-Since "";
+      proxy_set_header Range "";
+      proxy_set_header If-Range "";
```

Comment:
I think we should move these up a level so they are in the `server` block to make sure they are applied for all locations that might proxy.

Reply:
you're totally right